jetbrains-core/src/software/aws/toolkits/jetbrains/settings/DynamicResourcesConfigurable.kt

```diff
@@ -112,4 +132,22 @@ class DynamicResourcesConfigurable : BoundConfigurable(message("aws.settings.dyn
             }
         }
     }
+
+    companion object {
+        private const val INITIAL_INPUT = "AWS::"
+
+        private fun showTypeSuggestionBox(): String? = MessagesService.getInstance().showMultilineInputDialog(
+            project = null,
+            message = message("aws.settings.dynamic_resources_configurable.suggest_types.dialog.message"),
+            title = message("aws.settings.dynamic_resources_configurable.suggest_types.dialog.title"),
+            initialValue = "AWS::",
```

Comment:
Should this be localized?

Reply:
No - this refers to the start of a resource type.